File: felipe/Problema0084/problema0084.java
import java.util.Scanner;

public class problema0084 {
    public static void main(String[] args) {
        Scanner input = new Scanner (System.in);
        int a,b,c;

        System.out.print("Digite A: ");
        a = Integer.parseInt(input.nextLine());

        System.out.print("Digite B: ");
        b = Integer.parseInt(input.nextLine());

        System.out.print("Digite C: ");
        c = Integer.parseInt(input.nextLine());

        if( (a > b) && (a > c) ) {

            System.out.print("'A' é o maior número.");

        } else {

            if( (b > a) && (b > c) ) {

                System.out.print("'B' é o maior número.");

            } else {

                System.out.print("'C' é o maior número.");

            }
        }
    }
}

File: felipe/Problema0084/problema0084.py
a = int(input("Digite A: "))
b = int(input("Digite B: "))
c = int(input("Digite C: "))

if(a > b) & (a > c):
    print("'A' é o maior número.")
elif(b > a) & (b > c):
    print("'B' é o maior número.")
else:
    print("'C' é o maior número.")
